tests/system/action/meeting/test_clone.py

```diff
@@ -167,6 +167,80 @@ def test_clone_with_users(self) -> None:
             },
         )
 
+    def test_clone_with_ex_users(self) -> None:
+        self.test_models["meeting/1"]["user_ids"] = [1]
+        self.test_models["group/1"]["user_ids"] = [1]
+        self.set_models(
+            {
+                "user/1": {
+                    "group_$_ids": ["1"],
+                    "group_$1_ids": [1],
+                    "meeting_ids": [1],
+                },
+                "user/2": {
+                    "username": "exuser1",
+                    "submitted_motion_$_ids": ["1"],
+                    "submitted_motion_$1_ids": [1],
+                },
+                "motion/1": {
```

Comment:
Please add 2 more users, only with usernames, and add them via "admin_ids" and "user_ids" in the request-call. Check, if those were exported also.

For the users you added via submitter, motion etc. use different user_ids, especially not the 1, which is added via meeting.user_ids.

Reply:
admin_ids and user_ids users are *not* exported. Why should they be exported? I already have a test for for admin/user_ids users.

Okay, I use now 11.

Reply:
Added two users to the *ex users* test. (admin and user_ids)

Reply:
> admin_ids and user_ids users are _not_ exported. Why should they be exported? I already have a test for for admin/user_ids users.
> 
> Okay, I use now 11.

Yes, you are right, the admin_ids/user_ids will not be exported